src/main/webapp/WEB-INF/jsp/links/configLinks.jsp

```diff
@@ -0,0 +1,325 @@
+        }
+
+        var updateUrl = function(e) {
+            var form = document.getElementById('${n}_edit_form');
+            form.li.link.url = e.target.value;
+            form.li.querySelector('dd.url').textContent = e.target.value;
+        }
+
+        var updateGroups = function(e) {
+            var form = document.getElementById('${n}_edit_form');
+            var checked = form.querySelectorAll(".groups input[type='checkbox']:checked");
+            var groups = Array.from(checked).map(function(e) { return e.name; });
+            console.log(groups);
+            if (groups.length === 0) groups = ['Everyone'];
+            form.li.link.groups = groups;
+            form.li.querySelector('dd.groups').textContent = form.li.link.groups;
+        }
+
+        var dragstart = function(e) {
+            source = e.target;
+            e.dataTransfer.setData("text/plain", e.target.innerHTML);
+            e.dataTransfer.effectAllowed = "move";
+        }
+
+        var dragover = function(e) {
+            e.preventDefault();
+            e.dataTransfer.dropEffect = "move";
+        }
+
+        var dropped = function(e) {
+            e.preventDefault();
+            e.stopPropagation();
+            var li = parentLi(e.target);
+            var liLink = li.link;
+            var srcLink = source.link;
+            source.innerHTML = li.innerHTML;
+            source.link = liLink;
+            li.innerHTML = e.dataTransfer.getData("text/plain");
+            li.link = srcLink;
+
+            var app = document.getElementById('${n}config');
+            populateEditForm(app.querySelector('li.active'));
+        }
+
+        var removeLink = function(e) {
+            var li = parentLi(e.target);
+            li.parentNode.removeChild(li);
+
+            var app = document.getElementById('${n}config');
+            if (!app.querySelector('li.active')) {
+                populateEditForm(app.querySelector('li'));
+                app.querySelector('li').classList.add('active');
+            }
+        }
+
+        var beforeSave = function() {
+            var app = document.getElementById('${n}config');
+            var links = [];
+            _.forEach(app.querySelectorAll("li"), function(item) {
+                links.push(item.link);
+            });
+            app.querySelector('input[name="links"]').value = JSON.stringify(links);
+        }
+
+        $(document).ready(function() {
+            ${n}.links = JSON.parse('${linksJson}');
+
+            if (${n}.links.length === 0) {
+                ${n}.links.push(cloneDefaultLink());
+            }
+
+            var app = document.getElementById('${n}config');
+            _.forEach(${n}.links, function(link) {
+                var item = app.querySelector('ul').appendChild(createLinkListItem(link));
+            });
+            _.forEach(app.querySelectorAll("li"), function(item, i) {
```

Comment:
:art: replace `"` with `'` to remain consistent with quotes in the rest of the file